### DISCUSSION TITLE
Problem with vicon when with object twitching.

### DISCUSSION THREAD

**Discussion by AhmedBaza1** [USC-ACTLab/crazyswarm#515] [Q&A]
Hello, 

I'm flying Crazflie2.1 using the Crazyswarm on Ubuntu 18.04 with ROS melodic and vicon motion capture system. I got to the point where everything is working, the light ring is green. when I launch hover_swarm.launch + run hello_world.py the drone takes off and the land with no problem. 
The problem happens when the drone object in the motion capture disappears or twitches even for a fraction of a second. I get the flowing error massage from the launch file: 
`[crazyswarm_server-2] process has died [pid 31071, exit code -6, cmd /home/baza/crazyswarm/ros_ws/devel/lib/crazyswarm/crazyswarm_server __name:=crazyswarm_server __log:=/home/baza/.ros/log/3ef5023a-3819-11ec-9e85-9cb6d01d6ec1/crazyswarm_server-2.log].
log file: /home/baza/.ros/log/3ef5023a-3819-11ec-9e85-9cb6d01d6ec1/crazyswarm_server-2*.log
`
![12323](https://user-images.githubusercontent.com/85301677/139313044-92466e21-48d2-491b-b699-33b39c648643.png)

this wasn't a problem with the Crazyflie package with vicon_bridge  and the drones used to fly just fine. 
Vicon_bige used to stop publishing position for the time of losing track but then the done was able to recover again, here Crazswarm server returns an error and shuts down. 
is there a way to fix it or run Crazyswarm with vicon_bridg? 

Thank you so much in advance

**Comment by whoenig** [comment]
Strange - this does sound like a bug to me. It is not possible to run with vicon_bridge - vicon_bridge is essentially built-in to reduce latency. You can use `vrpn` instead of `vicon` as motion_capture_type and see if that helps.

Otherwise it would be great if you could provide a callstack from the crash. You would need to run with a debugger attached, see http://wiki.ros.org/roslaunch/Tutorials/Roslaunch%20Nodes%20in%20Valgrind%20or%20GDB for some hints.

**Comment by AhmedBaza1** [comment]
Sorry for thee delay in the reply, the project was hulted for a while, 
What I did was I checked the log and I traced the error to " Unknown regid body" error at crazyswarm server. I commented that line and when I tryed again, The motion capture was sending "NAN" for thetracked drone. 
what I found that solve this problem is when I used identical marker configuration for all the drones. it solved my issue at least for now
